rotkehlchen/tests/unit/test_cost_basis.py

```diff
@@ -1047,3 +1060,80 @@ def test_taxable_acquisition(accountant):
     assert pot.processed_events[0].free_amount == ZERO
     assert pot.processed_events[0].pnl.taxable == ONE
     assert pot.processed_events[0].pnl.free == ZERO
+
+
+@pytest.mark.parametrize('force_no_price_found_for', [[
```

Comment:
Why not just use simple mocked prices?

Reply:
Because we essentially don't need some prices for certain timestamps, but if there is no mocked price provided and it is not explicitly specified to raise an error, then there is an assertion error raised.

I thought that telling to raise an error is better than providing mocked prices because it shows that we in fact don't need these prices.

Reply:
No it's not. It's confusing to the reader, it's a much more complicated fixture and is not used as much in the code. And never used for this purpose. Please use the same mocked prices logic we use elsewhere.